{{ cookiecutter.project_shortname }}/.github/workflows/tests.yml

```diff
@@ -21,35 +21,82 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
+          # You can add/remove combinations e.g. dev requirements or postgresql13 by adding
+          # a new item to the following lists.
           python-version: [3.6, 3.7, 3.8, 3.9]
-          # You can add more combinations e.g. dev requirements or MySQL by adding
-          # a new item to this list. Add its corresponding definition below.
-          # EXTRAS and REQUIREMENTS_LEVEL are always needed.
-          services: [release, lowest, devel]
+          requirements-level: [min, pypi]
+          db-service: [postgresql9, postgresql11, mysql5, mysql8]
+          es-service: [es6, es7]
+          exclude:
+          # Add combinations to this list that should be excluded from the final
+          # build. Doing this will help keeping the number of jobs down.
+          # E.g. removing 3.8 - min combination will avoid 8 jobs to be submited
+          # [3.8] * [min] * [postgresql9, postgresql11, mysql5, mysql8] * [es6, es7]
+          - python-version: 3.8
+            requirements-level: min
+
+          - python-version: 3.9
+            requirements-level: min
+
+          - db-service: postgresql11
+            python-version: 3.6
+
+          - db-service: mysql8
+            python-version: 3.6
+
+          - search-service: es7
+            python-version: 3.6
+
+          - search-service: es6
+            db-service: postgresql11
+
+          - search-service: es6
+            db-service: mysql8
+
+          - search-service: es7
+            db-service: postgresql9
+
+          - search-service: es7
+            db-service: mysql5
+
           include:
-          - services: release
-            SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://invenio:invenio@localhost:5432/invenio"
-            POSTGRESQL_VERSION: POSTGRESQL_12_LATEST
-            ES_VERSION: "ES_7_LATEST"
-            EXTRAS: "all,postgresql,elasticsearch7"
-            REQUIREMENTS_LEVEL: "pypi"
-          - services: lowest
-            SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://invenio:invenio@localhost:5432/invenio"
+          - db-service: postgresql9
+            DB: postgresql
             POSTGRESQL_VERSION: POSTGRESQL_9_LATEST
-            ES_VERSION: "ES_6_LATEST"
-            EXTRAS: "all,postgresql,elasticsearch6"
-            REQUIREMENTS_LEVEL: "min"
-          - services: devel
             SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://invenio:invenio@localhost:5432/invenio"
-            POSTGRESQL_VERSION: POSTGRESQL_12_LATEST
-            ES_VERSION: "ES_7_LATEST"
+            EXTRAS: "all,postgresql"
+
+          - db-service: postgresql11
+            DB: postgresql
+            POSTGRESQL_VERSION: POSTGRESQL_11_LATEST
+            SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://invenio:invenio@localhost:5432/invenio"
+            EXTRAS: "all,postgresql"
```

Comment:
Major: Global `EXTRAS` cannot be set here because if there is a build that needs `postgresql11 ` and `es7`, then `es7`'s `EXTRAS` will be the only ones taken into account. Working on fixing this problem in https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/257.